src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/SupplementaryDatasetService.java

```diff
@@ -42,8 +42,16 @@ public void addSupplementaryDatasetEntity(SupplementaryDatasetDTO supplementaryD
       supplementaryDatasetRepository.save(supplementaryDatasetEntity);
       log.info("Successfully saved the supplementary dataset to the database");
     } catch (Exception e) {
-      throw new CTPException(
-          CTPException.Fault.SYSTEM_ERROR, "Something went wrong adding dataset", e);
+      if (collectionExercise == null) {
+        log.error(
+            "Failed to find collection exercise for supplementary dataset. "
+                + "SurveyId: {}, PeriodId: {}",
```

Comment:
Although the DTO fields for this object are `surveyId` and `periodId` (note the lowercasing), the values passed in the message are `survey_id` and `period_id`. As such we should keep the error message consistent and use those field names. It will avoid confusion and aid log searching.

Reply:
I have update the log messages to use the values `survey_id` and `period_id`